### ISSUE TITLE
any materials?

### ISSUE THREAD

**Issue by jeannotes** [lucasrm25/Multi-Object-Tracking#4]
any reference materials?

**Comment by lucasrm25** [comment]
Hi @jeannotes,

This project is related to the online course  **Multi-Object Tracking for Automotive Applications** on EDX:  https://www.edx.org/course/multi-object-tracking-for-automotive-systems-5

the course is online and is totally free (as soon as you don't want the certificate)

The equations and the whole material can be obtained when you register for the course. The equations are all rather complicated, so it is very hard to just follow by reading the code (you need the lecture notes).

In case you don't have experience with Kalman Filter, I suggest you the prerequisite course (also on EDX): https://www.edx.org/course/sensor-fusion-and-non-linear-filtering-for-automot

Cheers,
Lucas

**Comment by jeannotes** [comment]
@lucasrm25 actually, after I singed in, I didn';t get these materials, if I could have some papers or reference books, that would be better. can you suggest some?

**Comment by lucasrm25** [comment]
Indeed, the lecture slides were kind of "hidden" in the EDX platform. You can find them under the topic "Handouts" of each chapter.

I will send you the lecture notes by email.

**Comment by jeannotes** [comment]
@lucasrm25 email: j.z.feng@foxmail.com, I have learn the online videos, actually I believe I have understanding of kalman filters and related, but the course is hard to follow on. maybe later I have some questions to ask you, thanks!
